jumpscale/packages/marketplace/chats/meetings.py

```diff
@@ -0,0 +1,108 @@
+from jumpscale.sals.chatflows.chatflows import chatflow_step
+from jumpscale.sals.marketplace import MarketPlaceAppsChatflow, deployer
+from jumpscale.sals.reservation_chatflow import deployment_context, DeploymentFailed
+
+
+class MeetingsDeploy(MarketPlaceAppsChatflow):
+    FLIST_URL = "https://hub.grid.tf/tobias.3bot/jimbersoftware-freeflowconnect_grid-0.7.flist"
+    SOLUTION_TYPE = "meetings"
+    title = "Meetings"
+    steps = [
+        "get_solution_name",
+        "meetings_info",
+        "infrastructure_setup",
+        "reservation",
+        "initializing",
+        "success",
+    ]
+
+    query = {"cru": 1, "mru": 1, "sru": 0.5}
```

Comment:
this is the nginx query only, we should specify the workload resources + the nginx container resources.

Reply:
This is the exact same as the mattermost solution from which I copied a lot. Please explain further. https://github.com/threefoldtech/js-sdk/blob/b6efd79548528e4d2f82c0be5a84093a74ea5716/jumpscale/packages/marketplace/chats/mattermost.py#L19

Reply:
just like that comment https://github.com/threefoldtech/js-sdk/pull/1379#discussion_r501928635

we specify the query ( that's the total resources, main container + the nginx container )
then we have the main container resources, which we will use in the _deploy method

Please check the threebot example or this one https://github.com/threefoldtech/js-sdk/blob/490f800386a44ae1c8e4c668d6a2dfa2bd77fc0c/jumpscale/packages/marketplace/chats/taiga.py#L24